pkg/sys/stats_solaris.go

```diff
@@ -0,0 +1,41 @@
+/*
+ * Minio Cloud Storage, (C) 2016,2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sys
+
+import (
+	"os"
+)
+
+/*
+#include <unistd.h>
+*/
+import (
+	"C"
```

Comment:
We shouldn't use CGO based dependencies in Minio server. You should implement something Go native. @ptribble